src/sqsd.groovy

```diff
@@ -68,8 +68,8 @@ try {
         def messages = sqs.receiveMessage(receiveMessageRequest).getMessages()
         println "Received Messages : " + messages.size()
 
-        // Break when empty
-        if(messages.size() <= 0) break
+        // Break when empty if not running daemonized
+        if(messages.size() <= 0 && config.sqsd.run_daemonized < 1) break
```

Comment:
Sleeps because it won't break on empty queues. I guess it won't hurt to have that extra feature. I'd pull it after the configurable sleep and leave it off by default. We should later document this feature.